libsweep/examples/README.md

```diff
@@ -14,7 +14,7 @@ To build:
     mkdir build
     cd build
     cmake ..
-    cmake --build .
+    make
```

Comment:
Can you revert this change - cmake abstracts over the build system in use. On Linux the default generator is Unix Makefiles, so make is fine there. This is not the case everywhere. Also no the case when changing the generator, e.g. with `-G Ninja`.